### ISSUE TITLE
How to get the TSN

### ISSUE THREAD

**Issue by zengbohan0217** [hjeun/idu#4]
Hi, congrats for your work and great results!
From the article I knew that you used TSN as the feature extractor.We can download the TSN from the article you listed in your article, is that right?

**Comment by hyunjun-eun** [comment]
Yes, you can download [TSN](https://github.com/yjxiong/anet2016-cuhk) here.
